### PR TITLE
Removed all isPersonalised variables and methods, Add JavaDoc

### LINKED ISSUE #185: Include in assertions and javaDoc for PersonalMessageGenerator and DoneCommand

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

**Comment by yAOwzers** [comment]
Are there any other parts where we need to add JavaDocs to LOL

**Comment by franceneee** [comment]
Regarding InvalidModeException: 
As mentioned in the group, I think we should stick to throwing InvalidAppModeException. Both exceptions are thrown for the same reason (that the user is in the wrong mode). hehe, could you make that change please?

**Comment by yAOwzers** [comment]
Ok I've removed that exception

File: src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java [hunk 1]
```diff
@@ -30,6 +35,7 @@ public DoneCommandTimetableMode(String argument, AppState appState) {
     public void execute() throws ZeroNoteException {
         taskList = appState.getTaskList();
         int numberOfTasks = taskList.getNumberOfTasks();
+        assert numberOfTasks >= 0;
```
Comment (chuckiex3) [inline src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java:38]: nice!

Comment (franceneee) [inline src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java:38]: Nicely done!

**Review by franceneee — APPROVED** [review]

File: src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java [hunk 2]
```diff
@@ -10,6 +10,11 @@
 import zeronote.userinterface.CliMessages;
 import zeronote.userinterface.command.CliCommand;
 
+//@@author yAOwzers
+/**
+ * Marks a specific task in the tasklist in Zer0Note as done.
+ *
```
Comment (longngng) [inline src/main/java/zeronote/userinterface/command/timetable/DoneCommandTimetableMode.java:16]: I thought javadoc doesn't have this extra new line at the end, but it passed the test so i guess okay hehe.

**Review by longngng — APPROVED** [review]

**Pull request merged**